### PR TITLE
Unit Test Support

### PR DESCRIPTION
**Pull request by schineaj23**
Created complete tests for the DebugTable class as a template for possible future unit tests.

This also modifies the DebugTable class so that it returns a NetworkTableValue instead of a pure java Object. Every time we attempted to get() and cast, it threw a ClassCastException.

### CODE REVIEW

**Comment by schineaj23** [comment]
This addresses #11

File: src/main/java/frc/robot/utilities/DebugTable.java [hunk 1]
```diff
@@ -3,10 +3,11 @@
 import java.lang.Object;
 import edu.wpi.first.networktables.NetworkTable;
 import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
 
 public class DebugTable {
     static NetworkTable NetTabInst = NetworkTableInstance.getDefault().getTable("Debug");
-    public static Object get(String DataName) {
+    public static NetworkTableValue get(String DataName) {
         return NetTabInst.getValue(DataName);
```
Comment (Ayidana-Aboraah) [inline src/main/java/frc/robot/utilities/DebugTable.java:11]: Could call getValue again to return an object and then cast when used.

**Review by Ayidana-Aboraah — COMMENTED** [review]
I want to hear your feedback on the return type of DebugTable.get().

**Comment by schineaj23** [comment]
No longer relevant.

**Pull request closed without merging**